src/ncar-bufr2nc-fortran/gnssro_mod.f90

```diff
@@ -0,0 +1,581 @@
+!!!--------  README --------------------------------------------------------------------
+!  this is a temporary routine to generate netcdf file
+!  for jedi/ufo/gnssro/ operator test
+!  Copyright UCAR 2019
+!  Authors: Hailing Zhang
+!           Mark Olah
+
+!!!---------  to run   -----------------------------------------------------------------
+!  ./gnssro_bufr2ioda yyyymmddhh $bufrfile_input $netcdffile_output
```

Comment:
Remove this if it's a module not a program now. Does this mean we can remove the old converter then?